Jenkinsfile

```diff
@@ -17,31 +17,31 @@ pipeline {
     }
     stage('Build Go package') {
       steps {
-        sh './build.sh'
+        sh './build'
         archiveArtifacts artifacts: "dist/*.tar.gz,dist/*.zip,dist/*.rb,dist/*.deb,dist/*.rpm,dist/*.txt", fingerprint: true
       }
     }
     stage('Run unit tests') {
       steps {
-        sh './test.sh'
+        sh './test_unit'
         junit 'output/junit.xml'
       }
     }
 
     stage('Run acceptance tests') {
       steps {
-        sh 'cp ./dist/linux_amd64/summon summon'
-        dir('acceptance') {
-          sh 'make'
-        }
-        // TODO: remove need to sudo here
-        sh 'sudo chown -R jenkins:jenkins .'
-        // TODO: collect the acceptance test results
+        sh './test_acceptance'
+        junit 'output/acceptance/*.xml'
```

Comment:
artifact collection steps need to be wrapped in an `always` block, or if the previous `sh` step fails no results will be collected.